extract/tests/asos/femme-legging_test.py

```diff
@@ -0,0 +1,53 @@
+from core.constants import TABLE_NAME_SCRAPING_ASOS
+from core.domain import Product
+from extract import extract_product
```

Comment:
better to use relative import here

Reply:
The `core` package is a dependency. Since this code is in the extract package I mean this one: `from extract import extract_product`. It's better because of consistency. Normally we used relative imports.

Reply:
I am not sure if this is correct, but I think 'tests' is part of the **directory** 'extract', but it is **not** part of the 'extract' **package**, which is lying inside the extract directory. So using something like `from ..extract import extract_product` is not possible. In case of the zalando `t-shirt_test.py` we also just used `from extract import extract_product`, same goes for otto. In case I am wrong please tell me how it should be done.

Reply:
Ah, that's possible that it does not work because it's in the tests directory. If it does not work, ignore me 🙈 Sorry for this